src/components/RoomList/RoomList.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { ChangeEvent, MouseEvent } from 'react';
+import { CustomTableHeadProps, CustomTableProps, CustomTableStatefulVariables, CustomTableToolbarProps } from '../../common/types/TableTypes.types';
+import { CustomTable } from '../CustomTable/CustomTable';
+import { RoomData } from '../../common/types/TableTypes.types';
+import { RoomListHead } from './RoomListHead';
+import { RoomListToolbar } from './RoomListToolbar';
+import { SortingDirection } from '../../common/utils/SortingDirection';
+
+export function RoomList(): React.ReactElement<React.FC<CustomTableToolbarProps<RoomData>>> {
```

Comment:
This return type is not needed

Reply:
should be fixed in https://github.com/dorm-social-room-manager/dsrm-frontend/commit/4a08edda8a20f90d4793cad013f49e594ef837d6